commit 511136b4379faecd74fdcf01adab9a683b45a8f5
Author: David Sankel <david@stellarscience.com>
Date:   Tue Mar 26 16:13:01 2013 -0600

    Fixes that allow for compilation in -std=c++11 mode of gcc 4.7.2.
    
    In every case the problem was two double-quotes toget '""' in string literals which
    is no longer valid in C++ 2011.

Index: oce/src/Interface/Interface_Version.hxx
===================================================================
--- oce.orig/src/Interface/Interface_Version.hxx
+++ oce/src/Interface/Interface_Version.hxx
@@ -16,8 +16,8 @@
 // and conditions governing the rights and limitations under the License.
 
 #include <Standard_Version.hxx>
-#define XSTEP_PROCESSOR_VERSION "Open CASCADE %s processor "OCC_VERSION_STRING
-#define XSTEP_SYSTEM_VERSION    "Open CASCADE "OCC_VERSION_STRING
+#define XSTEP_PROCESSOR_VERSION "Open CASCADE %s processor " OCC_VERSION_STRING
+#define XSTEP_SYSTEM_VERSION    "Open CASCADE " OCC_VERSION_STRING
 #define XSTEP_Config            OCC_VERSION_STRING
 #define XSTEP_ULNames           OCC_VERSION_STRING
 
Index: oce/src/STEPControl/STEPControl_Controller.cxx
===================================================================
--- oce.orig/src/STEPControl/STEPControl_Controller.cxx
+++ oce/src/STEPControl/STEPControl_Controller.cxx
@@ -68,7 +68,7 @@ STEPControl_Controller::STEPControl_Cont
   if (!init) {
     RWHeaderSection::Init();  RWStepAP214::Init();
 
-    Interface_Static::Init ("step","write.step.product.name",'t',"Open CASCADE STEP translator "OCC_VERSION_STRING);
+    Interface_Static::Init ("step","write.step.product.name",'t',"Open CASCADE STEP translator " OCC_VERSION_STRING);
     Interface_Static::Init ("step","write.step.assembly",'e',"");
     Interface_Static::Init ("step","write.step.assembly",'&',"enum 0");
     Interface_Static::Init ("step","write.step.assembly",'&',"eval Off");
